cogeo_mosaic_tiler/handlers/app.py

```diff
@@ -266,40 +239,29 @@ def _geojson(mosaicid: str = None, url: str = None) -> Tuple[str, str, str]:
     binary_b64encode=True,
     tag=["tiles"],
 )
-@app.route(
-    "/<regex([0-9A-Fa-f]{56}):mosaicid>/tilejson.json",
-    methods=["GET"],
-    cors=True,
-    payload_compression_method="gzip",
-    binary_b64encode=True,
-    tag=["tiles"],
-)
 def _tilejson(
-    mosaicid: str = None,
-    url: str = None,
-    tile_scale: int = 1,
-    tile_format: str = None,
-    **kwargs: Any,
+    url: str = None, tile_scale: int = 1, tile_format: str = None, **kwargs: Any,
 ) -> Tuple[str, str, str]:
     """Handle /tilejson.json requests."""
-    if mosaicid:
-        url = _create_path(mosaicid)
-    elif url is None:
+    if url is None:
         return ("NOK", "text/plain", "Missing 'URL' parameter")
 
-    mosaic_def = fetch_mosaic_definition(url)
+    with MosaicBackend(url) as mosaic:
+        mosaic_def = dict(mosaic.mosaic_def)
+
+    return get_tilejson(mosaic_def, url, tile_scale, tile_format, **kwargs)
 
+
+def get_tilejson(mosaic_def, url, tile_scale, tile_format, **kwargs):
```

Comment:
`get_tilejson` could probably be refactored to return the tilejson dict, instead of `("OK", "application/json", json.dumps(meta))` directly.